client/src/app/components/chrysalis/TransactionPayload.tsx

```diff
@@ -28,7 +30,8 @@ class TransactionPayload extends AsyncComponent<TransactionPayloadProps, Transac
 
         this.state = {
             showInputDetails: -1,
-            showOutputDetails: -1
+            showOutputDetails: -1,
+            toggleBalance: false
```

Comment:
Should we rename this to something like `isFormattedBalance` ?

Reply:
Done.